### ISSUE TITLE
script to generate new blog post

### ISSUE THREAD

**Issue by garfieldnate** [garfieldnate/garfieldnate.github.io#1]
From the project readme:

>New blog posts go under `src/pages/blog/posts`. The name should be `YYYY-MM-DD-[slug].md`, and the frontmatter should contain the title, description, hero image src and alt text, the draft status, and the intended publication date.

Should be fairly easy to automate.
